kamailio/datadog_checks/kamailio/__about__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
```suggestion
__version__ = '1.0.0'
```

Reply:
resolved in cf85a7a2cd46cc4e6c399e639ff477102a24aead